EVENT.md

```diff
@@ -383,12 +383,12 @@ reflect a final set of parameters.
 
 When the browser receives an attribution trigger redirect on a URL matching a
 `destination` eTLD+1, it looks up all sources in storage that match
-<`attributionsrc` origin, `destination`> and picks the one with the greatest
+<reporting origin, `destination`> and picks the one with the greatest
```

Comment:
```suggestion
<reporting origin, `destination` eTLD+1> and picks the one with the greatest
```

Not sure if we should be usign `eTLD+1` or `site` as our terminology, but we should try to be consistent here and in `verbose_debugging_reports.md`.